rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
Thus the proposal to disable this check entirely for specs and revert to default (or line-count explicitly) otherwise.